source/directives/form.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference path="../../node_modules/forms-js/dist/forms-js.d.ts" />
+
+module adaptor.directives {
+
+  export function FormDirective($compile, builder) {
+    var formsjsForm:formsjs.Form;
+
+    return {
+      restrict: 'AE',
+
+      scope: {
+        view: '=',
+        validation: '=',
+        model: '='
+      },
+
+      controller: function() {
+        formsjsForm = new formsjs.Form();
+
+        this.form = formsjsForm;
+      },
+
+      link: function($scope:any, $element:ng.IAugmentedJQuery, $attributes:ng.IAttributes) {
+        $scope.$watch('model', function(model:any) {
+          formsjsForm.formData = model;
+        });
+
+        $scope.$watch('validation', function(validation:formsjs.ValidationSchema) {
```

Comment:
I haven't put much thought into this TBH. Off the top of my head, I think it would be reasonable for us *not* to bind here- and instead just passively/lazily act on changes to the validation rules unless the user explicitly told us to re-evaluate (perhaps via a public method we expose). This is how formFor does it at least. Deep changes to validation rules won't immediately trigger a re-evaluation (unless the user calls `form.validate()`) but they will be picked up automatically the next time a validation is run.